service/co/hotwax/shopify/app/ShopifyAppServices.xml

```diff
@@ -312,6 +312,9 @@ under the License.
             <parameter name="instanceAddress" required="true"/>
             <parameter name="instanceToken" required="true"/>
         </in-parameters>
+        <out-parameters>
+            <parameter name="error" required="false"/>
```

Comment:
why this is needed?
moqui support rest response code, do we still need this?

Reply:
Makes sense Sir. Updates the code. 
Initially, we considered that if the values are successfully stored at the keychain, it should be considered as successful irrespective of the API failure (which could be due to certain reasons like token expiration, instance down, etc), and if there is an error message it should be displayed.